modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -305,7 +305,25 @@ private synchronized void _generate(
 					ObjectFieldUtil.createObjectField(
 						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
 						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
-						_language.get(locale, "label"), "label", true)));
+						_language.get(locale, "name"), "name", true),
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_BOOLEAN,
+						ObjectFieldConstants.DB_TYPE_BOOLEAN, true, false, null,
+						_language.get(locale, "include"), "include", false),
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
+						_language.get(locale, "list-type-definition-id"),
+						"listTypeDefinitionId", false),
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_BOOLEAN,
+						ObjectFieldConstants.DB_TYPE_BOOLEAN, true, false, null,
+						_language.get(locale, "multiple"), "multiple", false),
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_CLOB, true, false, null,
+						_language.get(locale, "preselected-list-type-entries"),
+						"preselectedListTypeEntries", false)));
```

Comment:
We may want to name this more generically, for later support of other types of lists. Maybe just `preselectedValues`?

Reply:
That'd be fine, I think we would also need a "type" too which would determine how we read those values in that field. Or we create different definitions of DynamicFilter. PickListFitler vs VocabularyFilter?

Reply:
We have a similar situation with field `renderer`, where there are several types, `internal`, `clientExtention` and possibly `custom`. Each one has different unique identifier, mutually exclusive.

Using a `type` in this way is not the nicest model, as values have different meanings, but I don't think it is technically wrong. Preselected values have 1:1 relationship with filter entry. Alternative would be to have a new entity, something like `FDSFilterPreselectedValues`. It would have something like `id`, `type` and `value`.  FDSDateFilter would have `fdsFilterPreselectedValuesId`. That wouldn't be great either, but I don't think it would be wrong. I'd prefer type just for the sake of simplicity. I don't think that are any potential severe pitfalls, specially as the value is a leaf field. It might be different if with was something like an ID of a another entity.